src/proto.rs

```diff
@@ -652,43 +743,43 @@ pub struct AuthenticatorData {
     /// The optional attestation.
     pub(crate) acd: Option<AttestedCredentialData>,
     /// Extensions supplied by the device.
-    pub(crate) extensions: Option<serde_cbor::Value>,
-}
-
-impl AuthenticatorData {
-    /// get an extension value for an identifier
-    pub fn get_extension_for_identifier(&self, id: String) -> Option<&[u8]> {
-        self.extensions
-            .as_ref()
-            .map(|v| cbor_try_map!(v).ok())
-            .flatten()
-            .map(|map| map.get(&serde_cbor::Value::Text(id)))
-            .flatten()
-            .map(|v| cbor_try_bytes!(v).ok())
-            .flatten()
-            .map(|v| v.as_slice())
-    }
-}
+    pub(crate) extensions: Option<AuthenticatorExtensionsClientOutputs>,
+    // pub(crate) extensions: Option<serde_cbor::Value>,
+}
+
+// impl AuthenticatorData {
```

Comment:
Not sure that we need this anymore, but wasn't sure about its original motivation